app/src/main/res/layout/activity_main.xml

```diff
@@ -4,15 +4,224 @@
+        app:layout_constraintBottom_toTopOf="@id/button_five"
+        app:layout_constraintEnd_toEndOf="@id/button_five"
+        app:layout_constraintStart_toStartOf="@id/button_five" />
+
+    <Button
+        android:id="@+id/button_nine"
+        style="@style/button_style"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/activity_main_calculator_nine_text"
+        app:layout_constraintBottom_toTopOf="@id/button_six"
+        app:layout_constraintEnd_toEndOf="@id/button_six"
+        app:layout_constraintStart_toStartOf="@id/button_six" />
+
+    <Button
+        android:id="@+id/button_division"
+        style="@style/button_style"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/activity_main_calculator_division_text"
+        app:layout_constraintBottom_toTopOf="@id/button_multiply"
+        app:layout_constraintEnd_toEndOf="@id/button_multiply"
+        app:layout_constraintStart_toStartOf="@id/button_multiply" />
+
+    <Button
+        android:id="@+id/button_four"
+        style="@style/button_style"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/activity_main_calculator_four_text"
+        app:layout_constraintBottom_toTopOf="@id/button_one"
+        app:layout_constraintEnd_toEndOf="@id/button_one"
+        app:layout_constraintStart_toStartOf="@id/button_one" />
+
+    <Button
+        android:id="@+id/button_five"
+        style="@style/button_style"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/activity_main_calculator_five_text"
+        app:layout_constraintBottom_toTopOf="@id/button_two"
+        app:layout_constraintEnd_toEndOf="@id/button_two"
+        app:layout_constraintStart_toStartOf="@id/button_two" />
+
+    <Button
+        android:id="@+id/button_six"
+        style="@style/button_style"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/activity_main_calculator_six_text"
+        app:layout_constraintBottom_toTopOf="@id/button_three"
+        app:layout_constraintEnd_toEndOf="@id/button_three"
+        app:layout_constraintStart_toStartOf="@id/button_three" />
+
+    <Button
+        android:id="@+id/button_multiply"
+        style="@style/button_style"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/activity_main_calculator_multiply_text"
+        app:layout_constraintBottom_toTopOf="@id/button_subtraction"
+        app:layout_constraintEnd_toEndOf="@id/button_subtraction"
+        app:layout_constraintStart_toStartOf="@id/button_subtraction" />
+
+    <Button
+        android:id="@+id/button_one"
+        style="@style/button_style"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
```

Comment:
this is not right, try with that :
android:layout_width="0dp"
android:layout_height="0dp"
app:layout_constraintWidth_default="wrap"
app:layout_constraintHeight_default="wrap"
this set as wrap all this dimensions

Reply:
Solved!